backend/tournesol/views/video.py

```diff
@@ -32,7 +32,9 @@
         description="Retrieve a list of recommended videos, sorted by decreasing total score.",
         parameters=[
             OpenApiParameter("search"),
-            OpenApiParameter("language"),
+            OpenApiParameter("languages",
+                             description="Accepted languages separated by commas "
+                                         "(e.g. 'en,fr,de'). If empty, accept all languages"),
```

Comment:
_Side note about OpenAPI and Swagger_ (you can ignore it :stuck_out_tongue: ):

This parameter is currently typed as single string. 
![image](https://user-images.githubusercontent.com/4726554/146814356-7736e825-c50b-4e37-b7eb-4251f4d50f3a.png)
(Visible on http://localhost:8000/docs/#/video/video_list )


Ideally, we would like to describe that this parameter accepts multiple values using the options of `OpenAPIParameter`.  
However, this comma-separated representation is not the default one [supported by the OpenAPI standard](https://swagger.io/docs/specification/serialization/#query), so we would need to set `explode=False` which is [not currently supported](https://github.com/ferdikoomen/openapi-typescript-codegen/issues/654) by the library we are using to generate the client code used by the frontend.  

So once again, let's keep it simple. Your current description is perfect for what we want to achieve. We'll just need to handle this comma-separated format ourselves in the frontend code.